package/MDAnalysis/core/universe.py

```diff
@@ -369,9 +369,20 @@ def __init__(self, topology=None, *coordinates, all_coordinates=False,
             if callable(transformations):
                 transformations = [transformations]
             self._trajectory.add_transformations(*transformations)
-
+        to_guess = list(to_guess)
```

Comment:
```suggestion

        to_guess = list(to_guess)
```

It was good to have a line skipped here. It helps separate the function into logical steps.